cgnsutilities/cgnsutilities.py

```diff
@@ -624,6 +624,10 @@ def simpleOCart(self, dh, hExtra, nExtra, sym, mgcycle, outFile, comm=None):
             "cmax": 3.0,
         }
 
+        # Use user-defined options if provided
+        if userOptions is not None:
+            hypOptions.update(userOptions)
+
         # Run pyHyp
         from pyhyp import pyHyp
```

Comment:
I had no idea that this depends on pyHyp (of course it makes sense in retrospect), but we should probably update `setup.py`

Reply:
Also, L633 imports `mpi4py`, which I'm not sure if it's entirely necessary. Do we support running `cgns_utils` in parallel? We should either make this clear (documentation + updating `setup.py`), or remove the parallel feature.

Reply:
Could this cause problems for the build bot because of circular dependencies? pyHyp has cgnsUtilities as a testing dependency.

Also, cgnsUtilities itself does not run in parallel, but the pyHyp call does. This is only used for simpleOCart so I'm not sure if it's worth adding as a dependency.